sveltekit/src/components/TVChartContainer.svelte

```diff
@@ -0,0 +1,41 @@
+<!-- svelte-ignore a11y-missing-attribute -->
+<script>
+  import { widget } from "$lib/charting_library";
+  import { UDFCompatibleDatafeed } from "$lib/datafeeds/udf/src/udf-compatible-datafeed";
+  import { onMount } from "svelte";
+
+  function getLanguageFromURL() {
+    const regex = new RegExp("[\\?&]lang=([^&#]*)");
+    const results = regex.exec(window.location.search);
+    return results === null
+      ? null
+      : decodeURIComponent(results[1].replace(/\+/g, " "));
+  }
+
+  onMount(() => {
+    const widgetOptions = {
+      symbol: "AAPL",
+      // BEWARE: no trailing slash is expected in feed URL
+      datafeed: new UDFCompatibleDatafeed("https://demo_feed.tradingview.com"),
```

Comment:
Our latest examples have additional arguments for the demo datafeed.
```suggestion
      datafeed: new UDFCompatibleDatafeed(
						"https://demo_feed.tradingview.com",
						undefined,
						{
							maxResponseLength: 1000,
							expectedOrder: 'latestFirst',
						}
					),
```

The actual formatting of the suggest should be ignored (the tab spacing is probably incorrect).